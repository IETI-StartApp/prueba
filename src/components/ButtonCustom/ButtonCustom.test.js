import React from "react";
import {Button, CustomTypography} from "./index";
import '@testing-library/jest-dom';
import {shallow} from 'enzyme';


describe('should render button', () => {

    const wrapper = shallow(<Button onClick={() => console.log("HOLA")}>
        Start Investing
    </Button>)

    test('should render button', () => {
        expect(wrapper).toMatchSnapshot();
    })

})
describe('should render typography', () => {

    const wrapper = shallow(<CustomTypography text={'red'}>
        prueba
    </CustomTypography>)

    test('should render typography', () => {
        expect(wrapper).toMatchSnapshot();
    })

})
