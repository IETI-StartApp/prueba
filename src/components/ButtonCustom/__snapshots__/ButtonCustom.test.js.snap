// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render button should render button 1`] = `
<styled.button
  onClick={[Function]}
>
  <WithStyles(WithStyles(ForwardRef(Typography)))>
    Start Investing
  </WithStyles(WithStyles(ForwardRef(Typography)))>
</styled.button>
`;

exports[`should render typography should render typography 1`] = `
<WithStyles(ForwardRef(Typography))
  classes={
    Object {
      "root": "WithStyles(ForwardRef(Typography))-root-1 WithStyles(ForwardRef(Typography))-root-2",
    }
  }
  text="red"
>
  prueba
</WithStyles(ForwardRef(Typography))>
`;
